app/messages/_layout.email.html.erb

```diff
@@ -1,62 +1,116 @@
+<!DOCTYPE html>
 <html>
-<body style="background-color: #FFFFFF;">
+<head>
+  <meta name="viewport" content="width=device-width">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>Canvas Email</title>
```

Comment:
Good question. Some clients (Outlook, Mail for Windows 10) display the `<title>` if it's present. So this would be up to the Canvas team do decide on how you want to use it. 

Similarly, it's possible to include a hidden block of text near the top that email clients will display in the preview (think Gmail's preview or iOS Mail). If Canvas wanted to offer a preview of some kind, this might be useful, but these are much deeper questions than this Responsive pull can address.